user/cinnamon/template.py

```diff
@@ -0,0 +1,89 @@
+pkgname = "cinnamon"
+pkgver = "6.2.9"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Ddefault_library=shared"]
+hostmakedepends = [
+    "gobject-introspection",
+    "intltool",
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "at-spi2-core-devel",
+    "cinnamon-menus-devel",
+    "cjs-devel",
+    "dbus-devel",
+    "glib-devel",
+    "gstreamer-devel",
+    "gtk+3-devel",
+    "libx11-devel",
+    "libxml2-devel",
+    "mesa-devel",
+    "muffin-devel",
+    "networkmanager-devel",
+    "polkit-devel",
+    "xapp-devel",
+]
+depends = [
+    "bash",
+    "caribou",
+    "cinnamon-control-center",
+    "cinnamon-screensaver",
+    "cinnamon-session",
+    "libtimezonemap",
+    "mint-themes",
+    "nemo",
+    "network-manager-applet",
+    "python-pam",
+    "python-pexpect",
+    "python-pillow",
+    "python-pytz",
+    "python-requests",
+    "python-tinycss2",
+    "touchegg",
```

Comment:
add `accountsservice` and `gsound` here, cinnamon-settings doesn't launch without them for me